server/src-lib/Hasura/GraphQL/Transport/WebSocket/Server.hs

```diff
@@ -84,7 +103,7 @@ data WSConn a
   { _wcConnId    :: !WSId
   , _wcLogger    :: !(L.Logger L.Hasura)
   , _wcConnRaw   :: !WS.Connection
-  , _wcSendQ     :: !(STM.TQueue BL.ByteString)
+  , _wcSendQ     :: !(STM.TQueue (BL.ByteString, Maybe WSEventInfo))
```

Comment:
Instead of a tuple maybe we can create a separate ADT, maybe called `Response` or something like that? Which has the two fields.